src/org/project/Smartphone.java

```diff
@@ -0,0 +1,39 @@
+package org.project;
+
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Smartphone extends Product {
+    private Color color;
+    private int batteryCapacity;
+    private Accessories accessories;
```

Comment:
Hmmm, na pierwszy rzut oka to nie ma dla mnie sensu, jesli to ma byc jedno akcesorium to klasa i pole powinno byc nazwane jako liczba pojedyncza, jesli to ma byc kilka akcseriow to taka liczba mnoga tego nie obsluguje, wtedy pomyslalbym o liscie w tej warstwie (dla pojedynczych akcesoriow)